bin/ghe-backup

```diff
@@ -41,8 +41,13 @@ done
 # shellcheck source=share/github-backup-utils/ghe-backup-config
 . "$( dirname "${BASH_SOURCE[0]}" )/../share/github-backup-utils/ghe-backup-config"
 
+# Check to make sure moreutils parallel is installed
+ghe_parallel_check
```

Comment:
Ooops, I realise now I wasn't clear about what I had in mind in my comment earlier today. This approach would still need to add this check the `ghe-restore`.

What I had in mind was  is if you make this function, or even just its content, exec directly within `ghe-backup-config` you kill two birds with one stone.

Reply:
I replied to this comment below